internal/contentcache/contentcache.go

```diff
@@ -152,11 +151,21 @@ func (c *ContentCache) RecoverCache() error {
 		c.tempDir = "/tmp"
 	}
 	logger.Infof("Recovering cache:\n")
-	files, err := ioutil.ReadDir(c.tempDir)
+	dirEntries, err := os.ReadDir(c.tempDir)
 	if err != nil {
-		// if we fail to read the specified directory, log and return error
+		// We failed to get the list of sub-directories
+		// in the temp directory, log and return error.
 		return fmt.Errorf("recover cache: %w", err)
 	}
+	files := make([]os.FileInfo, len(dirEntries))
+	for i, dirEntry := range dirEntries {
+		files[i], err = dirEntry.Info()
+		if err != nil {
+			// We failed to read the specified directory,
```

Comment:
we are reading files inside the directory. either call it directory entry or file. read the specified directory is wrong word here.

Reply:
@vadlakondaswetha  Done. PTAL.

Reply:
@vadlakondaswetha ^